### PR TITLE
Icarus - Custom PCB

### PR DESCRIPTION
**Pull request by Babyyoda777**
Hi,
Hopefully I have sufficiently fixed the issues from before :) Changed it to private hardware and removed my protobufs.

### CODE REVIEW

File: variants/icarus/variant.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// Icarus has a 1.3 inch OLED Screen
+#define USCREEN_SSD106
```
Comment (caveman99) [inline variants/icarus/variant.h:2]: That looks like a typo. you sure?

File: variants/icarus/platformio.ini [hunk 2]
```diff
@@ -0,0 +1,19 @@
+[env:icarus]
+extends = esp32s3_base
+board = icarus
+board_level = extra
+board_check = true
+board_build.mcu = esp32s3
+upload_protocol = esptool
+upload_speed = 921600
+platform_packages = framework-arduinoespressif32@https://github.com/PowerFeather/powerfeather-meshtastic-arduino-lib/releases/download/2.0.16b/esp32-2.0.16.zip
```
Comment (caveman99) [inline variants/icarus/platformio.ini:9]: Is there something special about that framework? just curious.

**Review by caveman99 — APPROVED** [review]

File: variants/icarus/platformio.ini (hunk 2 above)
Comment (Babyyoda777) [reply variants/icarus/platformio.ini:9]: hi, yes it adds improved power management for esp32s

File: variants/icarus/variant.h (hunk 1 above)
Comment (Babyyoda777) [reply variants/icarus/variant.h:2]: it was in the seeedstudio esp32s3 variant.h and im using a similar screen, the firmware works so no issues there haha

Comment (caveman99) [reply variants/icarus/variant.h:2]: That is not really authoritative ... :-) 

I looked it up, that's definitely a typo. Should be SCREEN_SSD1306 (or any other screen model, these are usually auto detected) and you define them only if you have problem with autodetect. May be the reason it works for you despite the typo.

Comment (Babyyoda777) [reply variants/icarus/variant.h]: ah ok fixed :)

**Pull request merged**